src/components/common/DatePicker.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-form :attach="attach" ref="form" id="date-picker-form" class="date-time-selectors">
```

Comment:
It may be possible to have multiple "date picker forms" on the same page, so I wouldn't use an ID here (may be duplicate).

OTOH, the id would be a great name for the class instead :)

Reply:
hahah good point!

Reply:
Fixed!